src/common/os_dimm_windows.c

```diff
@@ -58,11 +58,16 @@ os_dimm_volume_handle(const char *path)
 	if (strncmp(path, UNC_PREFIX, UNC_PREFIX_LEN) == 0) {
 		path += UNC_PREFIX_LEN;
 	}
-	if (!GetVolumePathName(path, v, MAX_PATH - VOLUME_PREFIX_LEN)) {
+
+	if ((path[0] < 'a' || path[0] > 'z') &&
```

Comment:
I think in case of path[0] the if expression could be simplified by using "isalpha" function  ( see ctype.h  ) and checking the false result.